sgl-kernel/benchmark/bench_fp8_blockwise_gemm.py

```diff
@@ -9,7 +9,9 @@
 from sgl_kernel import fp8_blockwise_scaled_mm
 from vllm._custom_ops import cutlass_scaled_mm as vllm_scaled_mm
 
-from sglang.srt.layers.quantization.fp8_kernel import w8a8_block_fp8_matmul
+from sglang.srt.layers.quantization.fp8_kernel import (
+    w8a8_block_fp8_matmul_deepgemm as w8a8_block_fp8_matmul,
```

Comment:
```suggestion
    w8a8_block_fp8_matmul_triton as w8a8_block_fp8_matmul,
```
Ref: [this line](https://github.com/sgl-project/sglang/pull/6545/files#diff-30c0296c13657392b6438f93c0ac211d99b4f18e3aa87a5db4d893b3d667aa40R135)